package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "cesiumjs-workshop",
+  "version": "1.0.0",
+  "description": "Stater code for the CesiumJS Deep Dive Workshop, part of the 2025 Cesium Developer Conference.",
+  "homepage": "https://cesium.com/platform/cesiumjs/",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Cesium GS, Inc.",
+    "url": "https://cesium.com"
+  },
+  "keywords": [
+    "cesium",
+    "CesiumJS",
+    "vite",
+    "example",
+    "workshop",
+    "tutorial"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CesiumGS/cesiumjs-workshop.git"
+  },
+  "type": "module",
+  "scripts": {
+    "prepare": "husky install",
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "eslint": "eslint \"./**/*.js\" \"./**/*.html\" --cache --quiet",
+    "prettier": "prettier --write --no-config \"**/*\"",
+    "prettier-check": "prettier --check --no-config \"**/*\""
+  },
+  "devDependencies": {
+    "eslint": "^8.56.0",
+    "eslint-config-cesium": "^10.0.2",
+    "eslint-plugin-es": "^4.1.0",
+    "eslint-plugin-html": "^7.1.0",
+    "husky": "^8.0.3",
+    "prettier": "^3.2.4",
+    "vite": "^5.0.8",
+    "vite-plugin-static-copy": "^1.0.0"
+  },
+  "dependencies": {
+    "cesium": "1.130.0"
```

Comment:
@lukemckinstry Just a heads up: I locked the version of CesiumJS, just in case, to ensure this always works going forward.